gateway-spi/src/main/java/org/apache/knox/gateway/services/security/token/impl/JWTToken.java

```diff
@@ -122,6 +129,10 @@ public JWTToken(String alg, String[] claimsArray, List<String> audiences, boolea
     jwt = new SignedJWT(header, claims);
   }
 
+  private String getClaimValue(String[] claims, int index) {
+    return claims == null || claims.length < index ? null : claims[index];
```

Comment:
There seems to be an off by one in the validation. Shouldn't this be `<=` ?

```java
return claims == null || claims.length <= index ? null : claims[index];
```

Reply:
Nice catch! Fixed :)